main.tf

```diff
@@ -30,9 +30,23 @@ module "account" {
   sso_lastname             = var.account.sso_lastname
 }
 
+resource "aws_iam_policy" "workspace_boundary" {
+  provider = aws.account
+  count    = var.permission_boundaries.workspace_boundary_name != null ? 1 : 0
```

Comment:
the same count structure you did for workload_boundary can you do taht here? with the `&&`

Reply:
Done